api/src/websocket/utils/get-expires-at-for-token.ts

```diff
@@ -0,0 +1,9 @@
+import jwt from 'jsonwebtoken';
+
+export function getExpiresAtForToken(token: string): number | null {
+	const decoded = jwt.decode(token);
```

Comment:
I believe similarly in `getAccountabilityForToken`, should we only get the decoded `exp` if it's a Directus issued token? 🤔

Although this is very much an edge case and requires a user to be able edit their own static token, but it seems to be technically possible to use a generic JWT such as `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2ODE1NTQ4MDB9.91LkM-af_GOTpawYNPRbVK9dfqMJyA_J7UghPLEz8W8` (the payload is `{ "exp": 1681554800 }`) to "artificially" extend the auth timer duration.

Reply:
For this utility specifically it shouldn't matter where the JWT token comes from but the token should definitely be checked for validity before being handed to to this function.

The check for whether it is a valid Directus token happens in the `getAccountabilityForToken` function that will throw an error if not validated https://github.com/directus/directus/blob/12085bacbabacc57a046534236fd65189533bc80/api/src/websocket/controllers/base.ts#L88-L89

Reply:
> Wouldn't this technically be lowering the expire duration of a static token because they dont expire at all

Ahh this is definitely more accurate than what I was (wrongly) thinking! 

The concern about valid Directus token was only about trying not to "accidentally" extract `exp`. In that case, will that count as a potential issue? Though understandably this might be very unlikely for a user to just so happen to set a custom JWT _with exp_ as their static token. Feel free to resolve this if it's too much of an edge case, especially in terms of MVP 👍